dataframe/query_result_meta.go

```diff
@@ -12,9 +12,13 @@ type QueryResultMeta struct {
 
 	// Used in Explore to show limit applied to search result
 	Limit int64 `json:"limit,omitempty"`
+
+	// Visualization is so a Grafana visualization can be suggested with
+	// the response such as a graph or table.
+	Visualization string `json:"visualization,omitempty"`
```

Comment:
Could also be a struct with two fields, org/plugin and visualization. Where `""` is "grafana" as default. Then no string parsing of value.

Reply:
reverted vis property